eng/common/scripts/Invoke-DevOpsAPI.ps1

```diff
@@ -158,5 +158,26 @@ function Add-RetentionLease {
           -Headers (Get-DevOpsApiHeaders -Base64EncodedToken $Base64EncodedAuthToken) `
           -MaximumRetryCount 3 `
           -ContentType "application/json"
+}
+
+function Get-PackagesInArtifactFeed {
+  param (
+    $Organization="azure-sdk",
+    $Project="public",
+    [Parameter(Mandatory = $true)]
+    $FeedId,
+    $ApiVersion="6.1-preview.1",
+    $IncludeAllVersions="false",
+    $Base64EncodedAuthToken
+  )
+
+  $uri = "https://feeds.dev.azure.com/$Organization/$Project/_apis/packaging/Feeds/$FeedId/packages?api-version=$ApiVersion&includeAllVersions=$IncludeAllVersions"
```

Comment:
Did you see my other feedback in your .NET PR? If we can avoid it lets not try pulling all the package versions when we can just use the package version min and max to get the package we want.

Reply:
Or could you default to latest one if no feed version passed in?

Reply:
The challenge is to find the latest alpha version in the feed. Since sometime other versions other than alpha is listed as the latest thats why I need to retrieve all the abvailable versions.

Reply:
I can alternatively get all versions for the specific package, but I will have to call the api numerous times.

Reply:
@chidozieononiwu I'd like to better understand why my suggestion at https://github.com/Azure/azure-sdk-for-net/pull/25503#discussion_r765146174 doesn't work before we move forward with this new API.

Reply:
I tried it and it is very unreliable. I installes some packages and fails to install other which are clearly present in the feed. I can not say why these failures occur.

Reply:
I just posted some updates to that code in your other PR. Please give that a try.

Also I would like us to avoid taking a dependency on the devops api directly as I'd prefer we use general nuget feed apis so we could move to another nuget feed, l like nuget if we needed to.